warehouse/migrations/versions/14ad61e054cf_add_project_lifecycle_status.py

```diff
@@ -0,0 +1,81 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Add Project Lifecycle Status
+
+Revision ID: 14ad61e054cf
+Revises: b14df478c48f
+Create Date: 2024-06-26 20:30:52.083447
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "14ad61e054cf"
+down_revision = "b14df478c48f"
+
+
+def upgrade():
```

Comment:
Seems like this should be fast since we're adding nullable columns only, but taking a moment to pause and think twice is always good.

Reply:
Since this touches the often-used Project table, I wanted to be a bit more safe than having to scramble.

Do we have either CLI-execution metrics/timings, or specifically timing for migrations emitted somewhere, or is that something we should add?